### PR TITLE
Update README.md, add DataIsKey under "Projects using CVAT"

### PR DESCRIPTION
**Pull request by olympus999**
### Motivation and context
Helping users find annotation services for their projects. The main tool we use is CVAT.

### Checklist

- [X] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [X] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +163,9 @@ Other ways to ask questions and get our support:
 - [Onepanel](https://github.com/onepanelio/core) - Onepanel is an open source
   vision AI platform that fully integrates CVAT with scalable data processing
   and parallelized training pipelines.
+- [DataIsKey](https://dataiskey.eu/annotation-tool/) - DataIsKey uses CVAT as their prime data labeling tool to offer annotation services for projects of any size. 
+  You send Us the data you wish to label, and we will return high-quality annotations at the best price!
```
Comment (nmanovic) [inline README.md:167]:
```suggestion
```

**Review by nmanovic — DISMISSED** [review]
@olympus999 , thanks for sharing. I keep only the first sentence. Hope it works for you.

File: README.md [hunk 2]
```diff
@@ -163,6 +163,8 @@ Other ways to ask questions and get our support:
 - [Onepanel](https://github.com/onepanelio/core) - Onepanel is an open source
   vision AI platform that fully integrates CVAT with scalable data processing
   and parallelized training pipelines.
+- [DataIsKey](https://dataiskey.eu/annotation-tool/) - DataIsKey uses CVAT as their prime data labeling tool to offer annotation services for projects of any size. 
+
```
Comment (nmanovic) [inline README.md:167]:
```suggestion
```

**Comment by nmanovic** [comment]
@dvkruchinin ,

Could you please fix remark linter?

```sh
Error: Could not find reporter `json`
    at log (/home/runner/work/cvat/cvat/node_modules/unified-engine/lib/file-set-pipeline/log.js:16:12)
    at wrapped (/home/runner/work/cvat/cvat/node_modules/trough/wrap.js:25:19)
    at next (/home/runner/work/cvat/cvat/node_modules/trough/index.js:57:24)
    at FileSet.done (/home/runner/work/cvat/cvat/node_modules/trough/wrap.js:55:16)
    at FileSet.emit (events.js:314:20)
    at FileSet.one (/home/runner/work/cvat/cvat/node_modules/unified-engine/lib/file-set.js:108:10)
    at FileSet.emit (events.js:314:20)
    at done (/home/runner/work/cvat/cvat/node_modules/unified-engine/lib/file-set-pipeline/transform.js:54:15)
    at next (/home/runner/work/cvat/cvat/node_modules/trough/index.js:59:14)
    at done (/home/runner/work/cvat/cvat/node_modules/trough/wrap.js:55:16)
```

**Comment by dvkruchinin** [comment]
@nmanovic, 
> Could you please fix remark linter?

I prepared the PR #3157 for this.

File: README.md [hunk 3]
```diff
@@ -163,6 +163,7 @@ Other ways to ask questions and get our support:
 - [Onepanel](https://github.com/onepanelio/core) - Onepanel is an open source
   vision AI platform that fully integrates CVAT with scalable data processing
   and parallelized training pipelines.
+- [DataIsKey](https://dataiskey.eu/annotation-tool/) - DataIsKey uses CVAT as their prime data labeling tool to offer annotation services for projects of any size.
```
Comment (nmanovic) [inline README.md:166]:
```suggestion
- [DataIsKey](https://dataiskey.eu/annotation-tool/) uses CVAT as their prime data labeling tool to offer annotation services for projects of any size. 
```

File: README.md [hunk 4]
```diff
@@ -163,6 +163,7 @@ Other ways to ask questions and get our support:
 - [Onepanel](https://github.com/onepanelio/core) - Onepanel is an open source
```
Comment (nmanovic) [inline README.md:163]:
```suggestion
- [Onepanel](https://github.com/onepanelio/core) is an open source
```

**Review by nmanovic — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -160,9 +160,10 @@ Other ways to ask questions and get our support:
 
 ## Projects using CVAT
 
-- [Onepanel](https://github.com/onepanelio/core) - Onepanel is an open source
+- [Onepanel](https://github.com/onepanelio/core) is an open source
   vision AI platform that fully integrates CVAT with scalable data processing
   and parallelized training pipelines.
+- [DataIsKey](https://dataiskey.eu/annotation-tool/) uses CVAT as their prime data labeling tool to offer annotation services for projects of any size.
```
Comment (nmanovic) [inline README.md:166]:
```suggestion
- [DataIsKey](https://dataiskey.eu/annotation-tool/) uses CVAT as their prime data labeling tool
to offer annotation services for projects of any size. 
```

**Review by nmanovic — APPROVED** [review]

**Pull request merged**